packages/typo3-docs-theme/resources/public/css/theme.css

```diff
@@ -11156,7 +11156,7 @@ progress {
   height: auto;
 }
 
-.figure {
+.figure, figure {
```

Comment:
Oh boy, is this ugly. But okay for now, I think.

Reply:
This css is automatically generated...